main.cpp

```diff
@@ -393,11 +411,14 @@ struct Memory
 
 Memory::Memory()
 {
+    capacity = 12000;
+    lenght = 90;
     std::cout << "'Memory' created" << std::endl;
 }
```

Comment:
When you have to choose where to initialize your member variables and your choices are:
Constructor Member Initializer List
Inside the Constructor's body
Try to use the Constructor Member Initializer List.
The reason is that the member variables need to be instantiated/initialized when the class is created.
When you assign to them in the constructor body, you are potentially re-initializing them, which is a waste of resources.
Take a look at the output of this program: https://godbolt.org/z/q83YM63ze
Take note of the second instance being created and destroyed due to that constructor body assignment.
- [ ] revise, use the member initializer list to initialize your member variables
- [ ] Spell 'length' correctly.